testcases/OpTestGSBStaticKey.py

```diff
@@ -0,0 +1,299 @@
+        if int(output[0]) == 2: # Value '2' means Secure Boot enabled
+            return True
+        elif int(output[0]) == 0: # Value '0' means Secure Boot disabled
+            return False
+
+    def hmc_secureboot_on_off(self, enable=True):
+        '''
+        Enable/Disable Secure Boot from HMC
+        1. PowerOFF/Shutdown LPAR from HMC
+        2. Enable/Disable Secure boot using 'chsyscfg' command
+        3. PowerON/Activate the LPAR and boot to Operating System
+        '''
+        # PowerOFF/shutdown LPAR from HMC
+        self.cv_HMC.poweroff_lpar()
+        # Set Secure Boot value using HMC command
+        cmd = ('chsyscfg -r lpar -m %s -i "name=%s, secure_boot=' %
+               (self.cv_HMC.mg_system, self.cv_HMC.lpar_name))
+        if enable: # Value '2' to enable Secure Boot
+            cmd = '%s2"' % cmd
+        else: # Value '0' to disable Secure Boot
+            cmd = '%s0"' % cmd
+        # Execute the command
+        self.hmc_con.run_command(cmd, timeout=300)
+        # PowerON/Activate LPAR and Boot the Operating System
+        self.cv_SYSTEM.goto_state(OpSystemState.OFF)
+        self.cv_SYSTEM.goto_state(OpSystemState.OS)
+        # Checking What Secure Boot state got set,
+        # according to the required enable/disable
+        hmc_secureboot = self.check_hmc_secureboot_state()
+        if enable and not hmc_secureboot:
+                self.fail("HMC: Failed to enable Secure Boot")
+        elif not enable and hmc_secureboot:
+                self.fail("HMC: Failed to disable Secure Boot")
+
+    def getPRePDisk(self):
+        '''
+        Identify the PReP partition, this disk name is required to copy
+        signed grub. This manual step required for RHEL OS only.
+        '''
+        cmd = 'sfdisk -l | grep "PPC PReP Boot" | tail -1 | cut -d" " -f1'
+        out = self.c.run_command(cmd)
+        self.prepDisk = out[0]
+        if not self.prepDisk:
+            self.fail("%s: Failed to get PReP partition name" % self.distro)
+
+    def checkKernel(self):
+        vmlinux = "vmlinuz" # RHEL
+        if self.distro == 'sles':
+            vmlinux = "vmlinux"
+        # Checking whether the kernel is signed or not
+        cmd = "strings /boot/%s-$(uname -r) | tail -1" % vmlinux
+        out = self.c.run_command(cmd)
+        fail_msg = "%s - Kernel is not signed" % self.distro
+        self.assertIn(self.signature, out, fail_msg)
+
+    def checkGrub(self):
+        # Checking whether the grub is signed or not
+        cmd = "strings %s | tail -1" % self.grubFilename
+        out = self.c.run_command(cmd)
+        fail_msg = "Grub is not signed"
+        self.assertIn(self.signature, out, fail_msg)
+
+    def getRHELFiles(self):
+        '''
+        Get the signed grub file details, this is the manual step required for RHEL OS
+        '''
+        cmd = "rpm -ql grub2-ppc64le | grep core.elf"
+        out = self.c.run_command(cmd)
+        if 'core.elf' in out[0]:
```

Comment:
one help as we grep `core.elf` is this check required as I feel this condition always works

Reply:
Done.